.github/ISSUE_TEMPLATE.md

```diff
@@ -12,6 +12,8 @@
 
 ###Checklist:
 - [ ] I'm using Visual Studio Code Version [1.x.x] [Stable/Insider]
+- [ ] I'm using the extension version [1.x.x]
+- [ ] I've regared to the (FAQ)[https://github.com/robertohuertasm/vscode-icons/blob/master/README.md#faqs] and they did not help.
```

Comment:
I think you have another typo here regared => regarded :stuck_out_tongue_winking_eye: